components/scream/src/share/io/scorpio_output.cpp

```diff
@@ -106,8 +106,12 @@ void AtmosphereOutput::init()
 /* Overload the run routine to accept a TimeStamp or floating point for the input time */
 void AtmosphereOutput::run(const util::TimeStamp& time)
 {
-  // Pass the time in seconds and as a string to the run routine.
-  run_impl(time.get_seconds(),time.to_string());
+  if (m_filename_with_time_string) {
```

Comment:
Perhaps it would be easier to add a `run ()` method, without timestamp input? I don't want to give users the option of accidentally disabling time string in the filename during normal runs. Adding a `run()` interface is probably safer, in that the AD will _never_ call this method, ensuring that no file overwrite happens in production runs.

Reply:
but for the tests I want to run using this method I am running through the AD.